custom-recipes/recommendation-system-auto-collaborative-filtering/recipe.json

```diff
@@ -152,6 +152,40 @@
             "label": "Top n most similar to compute scores",
             "type": "INT",
             "defaultValue": 10
+        },
+        {
+            "type": "SEPARATOR",
+            "name": "expert_mode_sep",
+            "label": "Expert mode"
+        },
+        {
+            "name": "expert_mode",
+            "label": "Expert mode",
+            "type": "BOOLEAN",
+            "defaultValue": false
+        },
+        {
+            "type": "SEPARATOR",
+            "name": "separator_parameters",
+            "label": "Timestamp filtering",
+            "visibilityCondition": "model.expert_mode"
+        },
+        {
+            "name": "timestamps_column_name",
+            "label": "Timestamps column",
+            "allowedColumnTypes": [
```

Comment:
We should allow also float and int columns (timestamp are int) and users may want to use some other kind of ordering column

Reply:
2131b6b